src/main/java/catan/domain/model/game/GameBean.java

```diff
@@ -78,6 +78,10 @@
     @Column(name = "INITIAL_BUILDINGS_SET", unique = false, nullable = false)
     private String initialBuildingsSet;
 
+    //TODO: think about renaming of this field as it will be used only in preparation stage
+    @Column(name = "CURRENT_CYCLE_BUILDING_NUMBER", unique = false, nullable = true)
+    private int currentCycleBuildingNumber;
```

Comment:
what about rename this field to currentBuildingCycle? qoz I was a bit cunfusing and though that this field means the current building number that is going to be build in this cycle